client/packages/system/src/Item/Components/ItemOptionRenderer/ItemOptionRenderer.tsx

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 import { ItemRowWithStatsFragment, ItemStockOnHandFragment } from '../../api';
 import { ItemOption } from '../../utils';
+import { Tooltip } from '@common/components';
 
 export const getItemOptionRenderer =
```

Comment:
I think this also sneaked it, i guess you updathis this branch from develop ? Maybe i'll update 2581 from develop before reviewing, to reduce diff

Reply:
yeah, that's also not intended.
I did merge from `main` in order to get the button name changes that Adam had made. perhaps I should've just updated the translation manually and waited for the conflict later.

I based off 2581 because I wanted to get the chart changes in here, and that hadn't been merged yet.

Reply:
yep, has come in from `main`